src/store/Storage.h

```diff
@@ -11,11 +11,38 @@
 
 #include "base/RefCount.h"
 #include "store/forward.h"
+#include "store_key_md5.h"
+#include "http/RequestMethod.h"
```

Comment:
Alphabetical order please to minimize enforcement followups. The "http/" goes before "store/"

Reply:
Fixed.